tests/shared/helpers.py

```diff
@@ -28,6 +28,10 @@
 from tests.shared.paths import PROJECT_ROOT
 
 TensorType = TypeVar('TensorType')
+TEST_ROOT = Path(__file__).absolute().parents[1]  # https://github.com/PyCQA/pylint/issues/5783
+PROJECT_ROOT = TEST_ROOT.parent.absolute()  # pylint:disable=no-member
+EXAMPLES_DIR = PROJECT_ROOT / 'examples'
+GITHUB_REPO_URL = 'https://github.com/openvinotoolkit/nncf/'
```

Comment:
Is it used somewhere?

Reply:
Indeed, in `tests.shared.helpers.create_venv_with_nncf` when we try to install from a commit on github.

Reply:
I can't see the usage of `GITHUB_REPO_URL` in your PR. 
The same 4 lines are defined in the tests/shared/paths.py
Why not reuse them from this file?

Reply:
I get it now, that was a rebase artifact. Removed.